### ISSUE TITLE
Workload's labels doesn't follow the recommendation of labels from Kubernetes.

### ISSUE THREAD

**Issue by JhoLee** [goharbor/harbor-helm#1045]
tl;dr
Suggesting Harbor Helm Chart to follow the [Kubernetes Labels Recommendation](backward compatibility).
---
I'm using Harbor and many apps with helm. 
And I use their kubernetes labels usefully like;
 1) Collecting metrics with Prometheus.
 2) Managing workloads.

But I knew that Harbor Helm has not follwed Kubernetes' label recommendation. 
([More information](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/))

So I suggest Harbor to follow this recommendation. 
Or I will commit it by myself, and I'll request for review.

Thank you.

**Comment by ninjadq** [comment]
Hi @JhoLee , thank you for mention this, could you help to commit a PR to add this label to harbor-helm, we already have some similar labels but not follow the k8s recommendation, you should keep them and add the new ones. Because some user may depend those labels in production

**Comment by JhoLee** [comment]
@ninjadq  OK. I'll commit it with backward compatibility.

**Comment by JhoLee** [comment]
@ninjadq  I committed those labels with backward compatibility!

**Comment by martinnirtl** [comment]
Hello!! Any update on this?

**Comment by billabongrob** [comment]
Yeah, curious as well.  Seems like none of the deployments or statefulsets inherit any of these labels.  😢
